## 00_autoconf2.6.diff by Jonathan Riddell <jriddell@ubuntu.com>
## Description: Allow us to use autoconf2.6
--- a/admin/cvs.sh
+++ b/admin/cvs.sh
@@ -29,7 +29,7 @@ check_autotool_versions()
 {
 AUTOCONF_VERSION=`$AUTOCONF --version | head -n 1`
 case $AUTOCONF_VERSION in
-  Autoconf*2.5* | autoconf*2.5* ) : ;;
+  Autoconf*2.5* | autoconf*2.5* | autoconf*2.6* ) : ;;
   "" )
     echo "*** AUTOCONF NOT FOUND!."
     echo "*** KDE requires autoconf 2.52, 2.53 or 2.54"
@@ -44,7 +44,7 @@ esac
  
 AUTOHEADER_VERSION=`$AUTOHEADER --version | head -n 1`
 case $AUTOHEADER_VERSION in
-  Autoconf*2.5* | autoheader*2.5* ) : ;;
+  Autoconf*2.5* | autoheader*2.5* | autoheader*2.6* ) : ;;
   "" )
     echo "*** AUTOHEADER NOT FOUND!."
     echo "*** KDE requires autoheader 2.52 or 2.53 (part of autoconf)"
